frontend/src/pages/Q&A/Q&A.jsx

```diff
@@ -273,6 +269,9 @@ function Ques(props) {
                   >
                     👎 {item.downvote}
                   </button>
+                  <button>
+                    <Link to={`/getanswers/${item._id}`}>Get Answer</Link>
```

Comment:
instead to directing to a different page, make it like a pop-up, (check `Broadcast` page for reference)